android/assets/jsons/Civ V - Vanilla/UnitPromotions.json

```diff
@@ -52,14 +52,14 @@
 	{
 		"name": "Extended Range",
 		"prerequisites": ["Accuracy III","Barrage III","Targeting II","Bombardment II", "Wolfpack II"],
-		"effect": "+1 Range",
+		"effect": "+[1] Range",
```

Comment:
Put the + behind brackets, to enable - range. I think that promotions that trade off damage and range make for interesting player choices 😜